cli/command/builder/prune_test.go

```diff
@@ -21,8 +19,5 @@ func TestBuilderPromptTermination(t *testing.T) {
 		},
 	})
 	cmd := NewPruneCommand(cli)
-	test.TerminatePrompt(ctx, t, cmd, cli, func(t *testing.T, err error) {
-		t.Helper()
-		assert.ErrorIs(t, err, command.ErrPromptTerminated)
-	})
+	test.TerminatePrompt(ctx, t, cmd, cli, nil)
```

Comment:
The default is not checking for an error-type; are we losing coverage because of that?

```go
		if assertFunc != nil {
			assertFunc(t, err)
		} else {
			assert.NilError(t, err)
			assert.Equal(t, cli.ErrBuffer().String(), "")
		}
```